bedrock/firefox/templates/firefox/whatsnew/index-lite.id.html

```diff
@@ -31,16 +31,18 @@ <h1 class="c-page-header-up-to-date">{{ _('Your Firefox is up to date.') }}</h1>
   </header>
   <div class="lite-body">
     <div class="lite-content mzp-l-content">
-      <div class="lite-woman">
-        <img src="{{ static('img/firefox/whatsnew/lite/woman.png') }}" width="513" alt="">
+      <div class="lite-kv">
+        <img src="{{ static('img/firefox/whatsnew/lite/phone.png') }}" alt="">
       </div>
       <div class="lite-copy">
         <div class="lite-container">
-          <img src="{{ static('img/logos/firefox/logo-lite.png') }}" width="100" alt="">
-          <h1><img src="{{ static('img/logos/firefox/logo-lite-wordmark-white.png') }}" width="300" alt="Firefox Lite"></h1>
+          <div class="lite-heading">
+            <img src="{{ static('img/logos/firefox/logo-lite-2019.png') }}"  alt="" height="146" width="155">
+            <h1>The New <strong>Firefox Lite</strong></h1>
```

Comment:
I have a translation here for the title:

![image](https://user-images.githubusercontent.com/400117/67277310-724b1e80-f4be-11e9-86fd-6840ab9a1ffa.png)